src/utils.ts

```diff
@@ -349,6 +349,18 @@ export class Utils {
     }
   }
 
+  static getPositionContainerElement(el: HTMLElement): HTMLElement {
+    if (!el) return null;
+
+    const style = getComputedStyle(el);
+
+    if (style.position === 'relative' || style.position === 'absolute' || style.position === 'fixed') {
+      return el;
+    } else {
+      return this.getPositionContainerElement(el.parentElement);

```

Comment:
there is no 'this' in static methods. use return Utils.getPositionContainerElement(...)
also no need to have else {} (a bit more compact, although less readable)

Reply:
check for el.parentElement being undefined

Reply:
you still have this.xyz in a static method!